runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -731,6 +737,39 @@ private void highlightObjectsForLocation(final WorldPoint location, final int...
 		}
 	}
 
+	private void highlightObjectsInRegion(final String[] objectNames, final int[] regionIds)
```

Comment:
This is kind of a lot to be per tick (full scene scans of objects, with string comparisons...) maybe cache it?

Reply:
I've changed this to only do a scene tile scan when the clue is read and on plane change. This interacts with some weird client behavior (see the added field comments/commit body) but seems to be working fine.